### ISSUE TITLE
ICEBOX - 10-10 Short Form Phase 2: Expanding eligibility for Short Form

### ISSUE THREAD

**Issue by markfallows** [department-of-veterans-affairs/va.gov-team#40760]
## **UPDATE 9/13/2023**
>We are now getting into research for the Registration Only pathway, which will essentially use the Short Form flow for some of the scenarios.  We can leverage the functionality with some changes to accommodate the form for the various use cases.  The Registration Only pathway is taking the place of this expansion. I will close this epic.

## **UPDATE 8/16/2022**
>After meeting with Christopher Hawkins (HEC Leadership and SME for Service History), it was discovered that the Service History items on the application require additional verifications, and some require the financial information (Means test=financial assessment) to make priority group determinations.  **Therefore, we will not be pursuing this work right now.  There does not seem to be any service history that can be fast-tracked through the Short Form flow.**
>[8/16/22 Meeting Notes](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/application/va-application/product/Bi-weekly%20Stakeholder%20Mtg%20notes/8-16-2022%20Bi-weekly%20Health%20Care%20App%20Update.md)
>[Presentation review & notes](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/application/va-application/product/Bi-weekly%20Stakeholder%20Mtg%20notes/8-16-22%20Short%20Form%20-%20Expansion_%20Service%20History%20review.pptx)

## Background
Introduce the 10-10EZ Short Form to used based on various combats and/or exposures that Veterans may have experienced.  

This will not replace any existing methods of accessing the Short Form; only add to them.

We will also need to consider PACT Act eligibilities when it is passed
- Current list of Service History can be used to determine Short Form eligibility as it is today (no additional options)
     - Once the PACT Act is in place, Short Form eligibility may be implemented on additional list items
- **Some “fast track” statuses that we have listed already on the "Service History" page**
     - Former Prisoner of War
     - Discharged or Retired from the military for a disability incurred in the line of duty
     - Served in Southeast Asia during the Gulf War between August 2, 1990 and Nov 11, 1998
     - Served in Vietnam between January 9, 1962 and May 7, 1975
     - Served on active duty at least 30 days at Camp Lejeune from January 1, 1953 through December 31, 1987

<details>
<summary>Screenshot of current Service History page on EZ form</summary>

![image.png](https://user-images.githubusercontent.com/92328831/180050434-71d5f934-c7d7-4910-80e3-340b6a9f8d04.png)

</details>

# Checklist for Short Form - Expansion

## List your team, project contributors, and reviewers
<details>

### Product Team
- OCTO-DE Product Lead: Patrick Bateman
- Product Manager: Heather Justice, Mark Fallows
- FE Engineer:  Lisa Zapson
- BE Engineer: Lihan Li
- Designer:
- Researcher:
- Slack channel: #1010-health-apps
- GitHub tag: 1010-short-form

</details>

## List of project artifacts for requests

<details>

- [Project epic #40760](https://app.zenhub.com/workspaces/10-10-health-apps-5fff0cfd1462b6000e320fc7/issues/department-of-veterans-affairs/va.gov-team/40760)
- [10-10EZ Product outline](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/application/va-application/10-10EZ%20Health%20Care%20Application%20-%20Product%20Outline.md)
- [Initiative brief](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/application/va-application/Short%20Form/Short%20Form%20Expansion/ReadMe.md)
- Whiteboard sketch or other lo-fi prototypes or wireframes
- User flows
- Research plan 
- Conversation guide
- North Star and KPIs for the product
- Authenticated test-user logins, when applicable. **_Do not put staging credentials in your va.gov-team ticket; store or reference them in a .md file in the va.gov-team-sensitive repository_**
- Product URL(s)
     - va.gov/health-care/apply/application/introduction
- Use cases
- Finalized design prototype or mockup
- Content brief
- Regression test plans
- TestRail 
     - Test cases
     - Test plans
- Technical diagrams (architecture diagram, sequence diagram)
- Any new publicly-exposed endpoints
- Any new interactions with dependent VA backends
- Release plan
- Product guide for contact center

</details>
     
## List of process steps

<details>

- [ ] Create Initiative Brief with the Problem Statement and as many details known
     - [ ] Update the Initiative Brief after each step with links to the artifacts and more details known
- [ ] Discovery
     - [ ] What is the problem to be solved
     - [ ] What solutions have been made (internal or external) to resolve this problem
     - [ ] What are the possible solutions
     - [ ] How would they solve the problem
     - [ ] What are the efforts involved (time, design, technical)
     - [ ] What is the ROI potential for each solution
     - [ ] Does the solution require engagement with downstream system
          - [ ] Is there a data change, and can the downstream system accept the data?
- [ ] Create Epic and corresponding stories to align with the work needed (start with this list!)
- [ ] Initial design
     - [ ] Lo-Fi design ideas
     - [ ] Presentation and agreement with the team on direction
          - [ ] Confirm Frontend and Backend efforts
     - [ ] Presentation to Stakeholders and downstream system representatives
- [ ] Submit Collaboration Cycle request ticket
     - [ ] Review Collaboration Cycle pages
     - [ ] Start gathering artifacts (list above)
     - [ ] Maintain this checklist of artifacts, updating as they are shared
- [ ] Schedule Design Intent review - Refer back to the initial Collaboration Cycle Request ticket
     - [ ] Present design and obtain feedback
- [ ] Sign up for a Product Review session to present the initiative
- [ ] Request Content & IA review - Refer back to the initial Collaboration Cycle Request ticket
- [ ] Finalize design with feedback
     - [ ] Present to team, stakeholders and any other interested parties
-- [ ] Create prototype or basic working functionality in environments
     - [ ] The team can determine the best path, used for research
- [ ] Create Research Plan and Conversation Guide
- [ ] Schedule Midpoint Review - Refer back to the initial Collaboration Cycle Request ticket
- [ ] Request Research Review - Refer back to the initial Collaboration Cycle Request ticket
- [ ] Schedule research sessions with Perigean
- [ ] Conduct research sessions
     - [ ] Synthesize research findings
     - [ ] Review findings with the team
     - [ ] Make any necessary tickets/changes for design
- [ ] Create Use Cases
     - [ ] Identify test users
- [ ] Complete development
     - [ ] Engineers work with Designers on any questions/clarifications
     - [ ] Present to team for validation
     - [ ] Present to Stakeholders for validation
- [ ] Submit QA ticket and schedule with Tze (based on Dev ETA)
- [ ] Engage downstream system team for End-to-End QA coordination
     - [ ] Create E2E use cases and document results
     - [ ] Obtain signoff from downstream system team
- [ ] Request Analytics review - Refer back to the initial Collaboration Cycle Request ticket
- [ ] Complete QA and Accessibility QA
- [ ] Request Staging Review - Refer back to the initial Collaboration Cycle Request ticket
     - [ ] Review findings with the team
     - [ ] Create tickets for work to be completed before launch
- [ ] Create Usability and/or UAT research plan and conversation guide
- [ ] Request Research Plan review
- [ ] Schedule Usability/UAT sessions with Perigean
- [ ] Conduct Usability/UAT sessions
     - [ ] Synthesize research findings, if needed (not usually needed for UAT)
     - [ ] Review findings with the team
     - [ ] Make any necessary tickets/changes for design and/or development
- [ ] Complete any Design and/or Development work from Usability/UAT sessions
- [ ] Update Error matrix documentation with any new error states
- [ ] Update any FE and/or BE Engineering documentation
- [ ] Request Privacy, Security, Infrastructure readiness review - Refer back to the initial Collaboration Cycle Request ticket
- [ ] Update Contact Center guide
- [ ] Submit Contact Center guide review ticket
- [ ] Create Release Plan
     - [ ] Include E2E test results & signoff
- [ ] Conduct Launch Go/No Go with the team
     - [ ] Review this checklist
     - [ ] Review Release Plan
- [ ] Launch in a phased manner, according to Release Plan
- [ ] Continuous check-ins with downstream system
- [ ] Conduct Usability/UAT sessions
     - [ ] Synthesize research findings, if needed (not usually needed for UAT)
     - [ ] Review findings with the team
     - [ ] Make any necessary tickets for design and/or development
- [ ] Schedule or close all remaining tickets
     - Some tickets may be backlogged for future work, as they were not required to launch

</details>

**Comment by hdjustice** [comment]
After meeting with Christopher Hawkins (HEC Leadership and SME for Service History), it was discovered that the Service History items on the application require additional verifications, and some require the financial information (Means test=financial assessment) to make priority group determinations.  
[8/16/22 Meeting Notes](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/application/va-application/product/Bi-weekly%20Stakeholder%20Mtg%20notes/8-16-2022%20Bi-weekly%20Health%20Care%20App%20Update.md)
[Presentation review & notes](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/application/va-application/product/Bi-weekly%20Stakeholder%20Mtg%20notes/8-16-22%20Short%20Form%20-%20Expansion_%20Service%20History%20review.pptx)

**Comment by hdjustice** [comment]
Additional "Short Form" flow capability is being considered for Registration Only path
